### PR TITLE
fix(TemplateLibrary): Style for search box

### PR DESCRIPTION
**Pull request by abnazhor**
# Issue #183 
<OVERALL SUMMARY OF PULL REQUEST>

### Changes
- Modified style for search box.
  - As with button style, I've had to add style on styles.css as most of the style comes from React UI.
  - Modified style on React component (height). Not sure if measures are correct as I've had to adapt them myself.

### CODE REVIEW

**Comment by abnazhor** [comment]
@Michael-Grover Fixed!

**Comment by irmerk** [comment]
@abnazhor please @ me after @Michael-Grover reviews and you've addressed any further changes, wanting to test this locally after that.

File: src/styles.css [hunk 1]
```diff
@@ -8,4 +8,28 @@
...
+div.ui.input input, div.ui.input input::placeholder {
+    background-color: #141f3c;
+    border-radius: 3px;
+    color: #ffff;
+    opacity: 1 !important;
+    caret-color: #ffff !important;
+}
+
+div.ui.input input:focus {
+    background-color: #141f3c;
```
[the remark this answers is not in the document]
Comment (abnazhor) [reply src/styles.css]: Do you mean input when focused?

**Comment by abnazhor** [comment]
@Michael-Grover I don't think so, haven't changed anything related to font size.

File: src/styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (irmerk) [reply src/styles.css]: I don't think I understand the question. The background of the search bar should not be `#141f3c` no matter what, it should be up to who consumes this component. So if someone is using it outside the context of `template-studio-v2`, they should be able to have a different color.

[the remark this answers is not in the document]
Comment (abnazhor) [reply src/styles.css]: Oh, now I get it. Thank you!

**Comment by Michael-Grover** [comment]
@abnazhor for some reason, when I run this locally, the font size is smaller than the Netlify previews. I don't think it's related to your PR, so never mind.

**Comment by irmerk** [comment]
@abnazhor any progress on this?

**Comment by abnazhor** [comment]
All the issues were addressed a few days ago!

**Comment by irmerk** [comment]
I think we should still implement the change I mentioned here: https://github.com/accordproject/cicero-ui/pull/190/files#r339640448

**Comment by abnazhor** [comment]
That change has already been implemented adding a transparent background colour instead of a solid one, to remove any kind of solid background this input has, as you suggested.

File: src/styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (irmerk) [reply src/styles.css]: Background here is still not `transparent`

File: src/styles.css [hunk 2]
```diff
@@ -8,4 +8,28 @@
...
     opacity: 1 !important;
+}
+
+div.ui.input i {
+    color: #ffffff;
+    opacity: 1 !important;
+}
+
+div.ui.input input, div.ui.input input::placeholder {
+    background-color: #141f3c;
```
Comment (irmerk) [inline src/styles.css]: Background here is still not `transparent`

Comment (abnazhor) [reply src/styles.css]: Fixed!

Comment (irmerk) [reply src/styles.css]: You may not be pushing up the changes, because it has not been fixed yet. `#141f3c` should be `transparent`

Comment (abnazhor) [reply src/styles.css]: Oh! My bad, It's seems like I forgot to add that file...

**Comment by irmerk** [comment]
Closing as stale, feel free to come back to it @abnazhor!

**Comment by abnazhor** [comment]
But... It was already modified...

**Comment by irmerk** [comment]
Ah alright, sorry about that!

**Pull request merged**